libraries/AP_Camera/AP_Camera_Mount.cpp

```diff
@@ -55,7 +55,7 @@ bool AP_Camera_Mount::set_tracking(TrackingType tracking_type, const Vector2f& p
 {
     AP_Mount* mount = AP::mount();
     if (mount != nullptr) {
-        return mount->set_tracking(0, tracking_type, p1, p2);
+        return mount->set_tracking(get_mount_instance(), tracking_type, p1, p2);
```

Comment:
This all looks good.  I think with PR https://github.com/ArduPilot/ardupilot/pull/24346 being merged there's one more that needs to be added for the set_lens() method.